libraries/RC_Channel/RC_Channel.h

```diff
@@ -511,6 +516,22 @@ class RC_Channels {
         return get_singleton() != nullptr && (_options & uint32_t(Option::USE_CRSF_LQ_AS_RSSI)) != 0;
     }
 
+    bool clear_overrides_by_roll(void) const {
```

Comment:
I think these accessors and clear_overrides_by_rc can be made protected or private because they are only ever used from within RC_Channel**s**.

Do we need to have bits and accessors for roll, pitch, yaw and throttle or could we just have a single bit and accessor to cover them all?  It seems sufficient to just allow pilots to retake control if they change any of the 4.

Reply:
Separating them allows for more flexible development, but also increases risk.
My requirements can be met with a single bit.
I made it private and change to a single bit.